docs/release-notes/8.0.38.md

```diff
@@ -0,0 +1,17 @@
+# Percona Server for MySQL 8.0.38
+
+Due to a critical issue that could prevent the server from restarting after creating a large number of tables (8001 or more), MySQL has removed the 8.0.38 release. MySQL has released 8.0.39 shortly (22 days later) after MySQL 8.0.38. Percona has skipped the release of Percona Server for MySQL 8.0.38. The next release of Percona Server for MySQL is Percona Server for MySQL 8.0.39-30 which contains all bug fixes and contents of both MySQL 8.0.38 and MySQL 8.0.39.
```

Comment:
Due to a critical fix, MySQL Community Server 8.0.39 was released shortly (22 days later) after MySQL Community Server 8.0.38. Percona has skipped the release of Percona Server for MySQL 8.0.38. Percona Server for MySQL 8.0.39 contains all bug fixes and contents from MySQL Community Server 8.0.38 and MySQL Community Server 8.0.39.

Reply:
updated